packages/core/helper-plugin/lib/src/components/RelationsInput/components/RelationItem.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '@strapi/design-system/Box';
+import { Flex } from '@strapi/design-system/Flex';
+
+export const RelationItem = ({ relations, index, children, endAction, ...props }) => {
+  return (
+    <Box {...props}>
```

Comment:
Do we need that `Box` wrapper, since everything else is wrapped in `Flex` already?